addons/purchase_requisition_stock/models/stock.py

```diff
@@ -45,7 +45,7 @@ class Orderpoint(models.Model):
     def _quantity_in_progress(self):
         res = super(Orderpoint, self)._quantity_in_progress()
         for op in self:
-            for pr in self.env['purchase.requisition'].search([('state', '=', 'draft'), ('reference', '=', op.name)]):
+            for pr in self.env['purchase.requisition'].sudo().search([('state', '=', 'draft'), ('reference', '=', op.name)]):
```

Comment:
This search in a loop doesn't seem efficient, I think you can replace it with a _read_group(..., groupby=['reference'],...) outside the loop, no ?

Reply:
It's in alternative functionality now. We can remove this override that become useless